feature-libs/customer-ticketing/styles/_customer-ticketing-actions.scss

```diff
@@ -0,0 +1,16 @@
+cx-customer-ticketing-close,
```

Comment:
let's use placeholder selector to not put more technical debt in the future. 

Other features (that I did too 😓) should be using %selectorname 

https://sass-lang.com/documentation/style-rules/placeholder-selectors
https://sap.github.io/spartacus-docs/css-architecture/#placeholder-selectors

- Allows the use of @extend directive at a more root level than doing @extend .something
- we can skip things using our mechanism for skipping component styles

Reply:
You need to have things like this set `feature-libs/cart/base/styles/components/_index.scss`